torch/_prims/__init__.py

```diff
@@ -57,6 +57,8 @@
     "atanh",
     "cos",
     "cosh",
+    "bessel_j0",
```

Comment:
Alphabetically these entries go after the `bessel_i` functions

Reply:
Ooops :p